src/ZfcUser/Factory/Form/ChangeEmail.php

```diff
@@ -18,13 +18,8 @@ class ChangeEmail implements FactoryInterface
 {
     public function createService(ServiceLocatorInterface $formElementManager)
     {
-        if ($formElementManager instanceof FormElementManager) {
-            $sm = $formElementManager->getServiceLocator();
-            $fem = $formElementManager;
-        } else {
-            $sm = $formElementManager;
-            $fem = $sm->get('FormElementManager');
-        }
+        $fem = $formElementManager;
```

Comment:
and now it is registered in ServiceManager so this is not BC break but bug then. 

But my point is that the factory could be used with FormElementManager and ServiceManager but this change eliminates ServiceManager it assumes that only FormElementManager can create it. And as you can see here https://github.com/ZF-Commons/ZfcUser/blob/1.x/Module.php#L90 it is created in ServiceManager. so bug? Or I missed something?